app/views/idv/by_mail/enter_code/index.html.erb

```diff
@@ -13,6 +13,10 @@
 <% title t('idv.gpo.title') %>
 <% end %>
 
+<%= render AlertComponent.new(type: :warning, class: 'margin-bottom-3', text_tag: 'div') do %>
```

Comment:
I think we should remove the `text_tag` attribute here so that the text renders in the default paragraph tag. It's only needed in the alert below because we're rendering two paragraphs there.

```suggestion
<%= render AlertComponent.new(type: :warning, class: 'margin-bottom-3') do %>
```

Reply:
Thanks!